commands/guild/twitch-announcer.js

```diff
@@ -0,0 +1,443 @@
+            .setTitle(
+              user.data[0].display_name +
+                ' is playing ' +
+                Twitch_DB.get(message.guild.id).twitchAnnouncer.gameName
+            )
+            .addField('Stream Title:', streamInfo.data[0].title)
+            .addField('Currently Playing:', streamInfo.data[0].game_name, true)
+            .addField('Viewers:', streamInfo.data[0].viewer_count, true)
+            .setColor('#6441A4')
+            .setFooter(
+              'Changed Game',
+              'https://static.twitchcdn.net/assets/favicon-32-d6025c14e900565d6177.png' // Official icon link from Twitch.tv
+            )
+            .setImage(
+              streamInfo.data[0].thumbnail_url
+                .replace(/{width}x{height}/g, '1280x720')
+                .concat('?r=' + Math.floor(Math.random() * 10000 + 1)) // to ensure the image updates when refreshed
+            )
+            .setThumbnail(user.data[0].profile_image_url)
+            .setTimestamp();
+          if (user.data[0].broadcaster_type == '')
+            changedEmbed.addField('Rank:', 'BASE!', true);
+          else {
+            changedEmbed.addField(
+              'Rank:',
+              user.data[0].broadcaster_type.toUpperCase() + '!',
+              true
+            );
+          }
+
+          //Game Change Edit
+          await announcedChannel.lastMessage.edit(changedEmbed);
+        }
+
+        //Offline Trigger
+        if (statusCheck == 'offline') {
+          Twitch_DB.set(`${message.guild.id}.twitchAnnouncer.status`, 'end');
+          const offlineEmbed = new MessageEmbed()
+            .setAuthor(
+              `Twitch Announcement: ${user.data[0].display_name} Offline`,
+              user.data[0].profile_image_url,
+              'https://twitch.tv/' + user.data[0].display_name
+            )
+            .setTitle(
+              user.data[0].display_name +
+                ' was playing ' +
+                Twitch_DB.get(message.guild.id).twitchAnnouncer.gameName
+            )
+
+            .setColor('#6441A4')
+            .setTimestamp()
+            .setFooter(
+              'Stream Ended',
+              'https://static.twitchcdn.net/assets/favicon-32-d6025c14e900565d6177.png'
+            )
+            .setThumbnail(user.data[0].profile_image_url);
+          if (!user.data[0].description == '')
+            offlineEmbed
+              .addField('Profile Description:', user.data[0].description)
+
+              .addField('View Counter:', user.data[0].view_count, true);
+          if (user.data[0].broadcaster_type == '')
+            offlineEmbed.addField('Rank:', 'BASE!', true);
+          else {
+            offlineEmbed.addField(
+              'Rank:',
+              user.data[0].broadcaster_type.toUpperCase() + '!',
+              true
+            );
+          }
+
+          //Offline Edit
+          try {
+            announcedChannel.lastMessage.edit(offlineEmbed);
+          } catch {}
```

Comment:
a catch block shouldn't be empty

Reply:
Fixed

yeah, i'm ashamed of that one (it was a poor way of handling the error that new configs caused)